src/main/java/Carrot.java

```diff
@@ -0,0 +1,219 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Carrot {
+    public static void main(String[] args) {
+        String logo = "   _____                     _   \n" +
+                        "  / ____|                   | |  \n" +
+                        " | |     __ _ _ __ _ __ ___ | |_ \n" +
+                        " | |    / _` | '__| '__/ _ \\| __|\n" +
+                        " | |___| (_| | |  | | | (_) | |_ \n" +
+                        "  \\_____\\__,_|_|  |_|  \\___/ \\__|\n";
+        System.out.println("Hello from\n" + logo);
+
+        // Chatbot functionality
+        greetUser();
+        printHelpCommand();
+        handleUserInput();
+    }
+
+    private static ArrayList<Task> listOfTasks = Storage.loadListOfTasks();
+
+    private static final String MESSAGE_DIVIDER = "-------------------------------------";
+    private static final String COMMAND_LIST =
+            "Available Commands:\n" +
+                    "1. todo <taskdescription> - Add a new todo task\n" +
+                    "2. deadline <taskdescription> /<by> - Add a new task with a deadline <by>\n" +
+                    "3. event <taskdescription> /<from> /<to> - Add a new task that starts <from> and ends <to>\n" +
+                    "4. mark <taskindex> - Mark a task as done\n" +
+                    "5. unmark <taskindex> - Mark a task as not done\n" +
+                    "6. delete <taskindex> - delete a task from the list\n" +
+                    "7. list - List all tasks recorded\n" +
+                    "8. bye - Exit the program\n" +
+                    "9. help - Show available commands";
+
+    private static void greetUser() {
+        System.out.println(MESSAGE_DIVIDER);
+        System.out.println("Hello! I'm Carrot!");
+        System.out.println("What can I do for you?");
+        System.out.println(MESSAGE_DIVIDER);
+    }
+
+    private static void handleUserInput() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String userInput = scanner.nextLine().trim();
+            CommandType userCommand = CommandManager.getCommandType(userInput);
+
+            processUserCommand(userCommand, userInput, scanner);
+        }
+    }
+
+    private static void processUserCommand(CommandType userCommand, String userInput, Scanner scanner) {
+        switch (userCommand) {
+        case HELP:
+            printHelpCommand();
+            break;
+        case LIST:
+            printListItems();
+            break;
+        case TODO:
```

Comment:
As per coding standards, if there is no break in the case statement, '// Fallthrough' comment should be added.
![image](https://github.com/nus-cs2113-AY2324S2/ip/assets/69302316/d0675607-1289-463a-b316-e09df1c432fa)